src/frontend/AppContainer.tsx

```diff
@@ -1,5 +1,144 @@
-import { createAppContainer } from "react-navigation";
+import * as React from "react";
+import debug from "debug";
+import {
+  createAppContainer,
+  NavigationActions,
+  NavigationContainerComponent,
+  NavigationState,
+} from "react-navigation";
+import AsyncStorage from "@react-native-community/async-storage";
 
+import { URI_PREFIX } from "./constants";
+// import useProjectInviteListener from "./hooks/useProjectInviteListener";
+import IS_E2E from "./lib/is-e2e";
+import bugsnag from "./lib/logger";
 import { WithModalsStack } from "./NavigationStacks/WithModalsStack";
 
-export default createAppContainer(WithModalsStack);
+// Turn on logging if in debug mode
+if (__DEV__) debug.enable("*");
+const log = debug("mapeo:App");
+// WARNING: This needs to change if we change the navigation structure
+const NAV_STORE_KEY = "@MapeoNavigation@9";
+const ERROR_STORE_KEY = "@MapeoError";
+
+const EDITING_SCREEN_NAMES = [
+  "AddPhoto",
+  "Camera",
+  "CategoryChooser",
+  "ManualGpsScreen",
+  "ObservationDetails",
+  "ObservationEdit",
+];
```

Comment:
This list will inevitably grow but these screens stood out as the ones where we'd want the invite modal to not show up